client/store.js

```diff
@@ -24,7 +24,7 @@ var events = new EE;
 // Private data
 var data = {
     categories: [],
-    categoryLabel: new Map(), // maps category ids to labels
+    categoryProperties: new Map(), // maps category ids to an object with two keys: label and color
```

Comment:
Ha, I hadn't seen this when I wrote my comment above. Let's just rename this `categoryMap` and have it map category id to categories, as suggested.